src/nodes/comment.gd

```diff
@@ -0,0 +1,33 @@
+tool
+extends ConceptNode
+
+
+func _init() -> void:
+	unique_id = "comment"
+	display_name = "Comment"
+	category = "Utilities"
+	description = "Insert comment in concept graph editor"
+	resizable = true
+
+func export_custom_data() -> Dictionary:
+	return {"comment_text": $LineEditComment.text}
+
+func restore_custom_data(data: Dictionary) -> void:
+	if not data.has("comment_text"):
+		return
+	$LineEditComment.text = data["comment_text"]
+
+func restore_editor_data(data: Dictionary) -> void:
+	.restore_editor_data(data)
+	# we need to emit resize request signal to restore rect size
+	if data.has("rect_x") && data.has("rect_y"):
+		emit_signal("resize_request", Vector2(data["rect_x"], data["rect_y"]))
+
+func has_custom_gui() -> bool:
+	return true
+
```

Comment:
Last thing, could you add an extra blank line between the functions? I'm trying to stick to the official gdscript guidelines https://docs.godotengine.org/en/stable/getting_started/scripting/gdscript/gdscript_styleguide.html#gdscript-style-guide

Reply:
Done